tests/tests/client.py

```diff
@@ -147,6 +147,33 @@ def addDeviceToGroup(self, group, device, expected_error=False, JWT="foo.bar.baz
             return r
 
 
+class ManagementClientV2:
+    log = logging.getLogger("Client")
+
+    def __init__(self, host, spec, api):
+        config = {
+            "also_return_response": True,
+            "validate_responses": True,
+            "validate_requests": True,
+            "validate_swagger_spec": True,
+            "use_models": True,
+        }
+
+        http_client = RequestsClient()
+        http_client.session.verify = False
+
+        self.client = SwaggerClient.from_spec(
+            load_file(spec), config=config, http_client=http_client,
+        )
+        self.client.swagger_spec.api_url = (
+            "http://%s/api/management/v2/inventory/" % host
```

Comment:
nit pick dont we have it defined in some cost somewhere? (I have not checked)

Reply:
with openapi it's completely different, but bravado is a bit clunky, and this is what we do in other places where we need v2 API